tests/test_modeling_lxmert.py

```diff
@@ -550,3 +549,139 @@ def test_model_from_pretrained(self):
         for model_name in LXMERT_PRETRAINED_MODEL_ARCHIVE_LIST[:1]:
             model = LxmertModel.from_pretrained(model_name)
             self.assertIsNotNone(model)
+
+    def test_attention_outputs(self):
```

Comment:
Needed to re-implement that test here, as well as the hidden states test since the behavior is different to other single-transformer models.